ujson/src/ujson/Value.scala

```diff
@@ -1,13 +1,9 @@
 package ujson
 
-
-
-import upickle.core.Util
-import upickle.core.{ObjArrVisitor, Visitor}
-
+import upickle.core._
```

Comment:
Can we keep explicit imports?

Reply:
`import upickle.core.{LinkedHashMap, ObjArrVisitor, Util, Visitor}` seems a bit too long, no?

Reply:
Not to me. I love when I can read source code without an IDE, which is not safely possible when there are multiple wildcard imports.

Reply:
This is even more true, when a type like `LinkedHashMap` also exists in the standard lib. A quick look at the imports shows where it really comes from.